File: java/src/main/java/itp/itp1_4/ITP1_4_A.java
package itp.itp1_4;

import java.util.Scanner;

public class ITP1_4_A {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    int a = sc.nextInt();
    int b = sc.nextInt();

    System.out.printf("%d %d %f\n", a / b, a % b, 1.0 * a / b);
  }
}

File: kotlin/src/itp/itp1_4/ITP1_4_A.kt
package itp.itp1_4

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val a = sc.nextInt()
    val b = sc.nextInt()
    println("${a/b} ${a%b} ${"%.5f".format(a.toDouble()/b)}")
}